packages/nimble-components/src/rich-text/specs/README.md

```diff
@@ -473,6 +896,43 @@ _Keyboard navigation with toolbar buttons focused_
 
 _Note_: All other keyboard interaction determined by the slotted element will not be defined in this document.
 
+_Keyboard interactions for `@mention`_
+
+| Key                                    | Behavior                                                                                                                            |
+| -------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------- |
+| `@`                                    | To open the list of options with the users list                                                                                     |
+|                                        | ---------------------------------------------Keyboard interactions when popup is opened-------------------------------------------- |
+| Any character                          | To show the filtered list of users for that specific character                                                                      |
+| Group of characters                    | To show the filtered list of users for the group characters                                                                         |
+| Group of characters(Not in any option) | Close the dropdown list popup                                                                                                       |
+| Enter, Tab                             | To select the currently focused option from the list                                                                                |
+| Up/Down Arrow keys                     | To move the focus upward/downward in the list of options                                                                            |
+| Left/Right Arrow keys                  | To move the cursor in the editor leftward/rightward and filters the list for the characters from `@` to the current cursor position |
```

Comment:
This behavior is the default behavior of Tiptap and here is the [link](https://tiptap.dev/api/nodes/mention#usage) to demonstrate this behavior.
![mention-left-right-arrow](https://github.com/ni/nimble/assets/123377523/10873ac2-9829-43b7-b0a5-f29e76ca35ea)

We thought of adopting the same behavior that Tiptap offers. This also provides a more precise way to filter the options for each particular interaction we perform inside the `@mention`.

Of course, other editors like AzDo or GitHub mention do not work like that. So we would like to align with you all about the accessibility for the specified keydown events.

If updating the lists for left/right arrow keystrokes is not the expected behavior, we might have to write our own implementations inside the `onUpdate()` method within the [render](https://tiptap.dev/api/utilities/suggestion#render) functionality to achieve what we wanted. @m-akinc Please let us know your thoughts.

Reply:
I will keep this comment open to get others' thoughts about the accessibility of the left/right arrow key behavior that Tiptap offers. If others don't have a concern about this, we will use the default behavior.

Reply:
I'm not sure this is an "accessibility" problem so to speak, as I don't believe there is any specific ARIA guidance to what the left and right arrow buttons should do with respect to a filtered list. I think is more of something @leslieab should weigh in on, but fwiw, I'm in agreement with @m-akinc, in that this feels weird to me for what feels like little value.

Reply:
The ARIA filterable combobox pattern includes using the left/right arrows on the menu to move visual focus to the field input and insert the caret one character to the left or right, and that is included in the IxD combobox spec.

I have no clue on whether what Tiptap is doing has accessibility considerations or not, but it is a very delightful microinteraction. It would be cool to have this in all our filterable lists, but since we don't we should err on the side of consistency, and decide whether this is an interaction we want to implement on other applicable components in the future.